src/ResourceManager/Network/Commands.Network/help/Get-AzureRmNetworkSecurityGroup.md

```diff
@@ -43,7 +43,7 @@ This command returns contents of Azure network security group "nsg1" in resource
 The credentials, account, tenant, and subscription used for communication with azure.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
+Type: IAzureContextContainer
```

Comment:
You'll need to regenerate the help files.

Remember to use the flag `-UseFullTypeName` when you generate the help files. instructions [here](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-a-single-markdown-file).